open_api_framework/conf/base.py

```diff
@@ -507,6 +527,11 @@
             "level": "DEBUG",
             "propagate": True,
         },
+        "celery": {
```

Comment:
While this will add the logs of the celery library itself, does the logging that happens inside celery tasks still work as well? Because from Sjoerd's issue I see that there is no logging at all. The regular logging is picked up by the project handler i think, even if it happens inside the celery worker

Reply:
It does appearing in Celery yes

![Screenshot from 2024-11-29 15-17-21](https://github.com/user-attachments/assets/c183cddb-9a40-4bcf-943b-9d2094a77d11)



Also with just INFO

![Screenshot from 2024-11-29 15-16-35](https://github.com/user-attachments/assets/c9dd96bd-90ea-4cff-9edc-778cb142fc5f)

Reply:
It does need `LOG_STDOUT` to be enabled though

Reply:
Check, I think it's fine to accept these changes then. I do think that the logs no longer showing has to do with either `LOG_STDOUT` or `LOG_LEVEL`, so I've asked Sjoerd to see what the values of those are https://github.com/open-zaak/open-zaak/issues/1802#issuecomment-2507964934